dimagi/commcare-hq
# Make MINIMUM_ZXCVBN_SCORE a setting rather than hard coding it into the logic

## Summary

This doesn't actually change anything, it was just annoying me that the minimum acceptable [zxcvbn](https://github.com/dropbox/zxcvbn) score for CommCare was hardcoded in inconsistent ways in 3-4 places.

## Feature Flag
None

## Product Description


## Safety Assurance


### Automated test coverage

Not sure yet.

### QA Plan

Manually test:
- Sign up new web user
- Create new mobile worker
- Reset webworker's own password
- Reset mobile worker's password

and make sure that it's respecting the `MINIMUM_ZXCVBN_SCORE` by testing it twice: once at 2 and once at 3.

### Safety story
I'll test this better before merging.

### Rollback instructions
